go/vt/vttablet/onlineddl/executor.go

```diff
@@ -942,12 +943,43 @@ func (e *Executor) cutOverVReplMigration(ctx context.Context, s *VReplStream, sh
 			// still have a record of the sentry table, and gcArtifacts() will still be able to take
 			// care of it in the future.
 		}()
-		parsed := sqlparser.BuildParsedQuery(sqlCreateSentryTable, sentryTableName)
-		if _, err := e.execQuery(ctx, parsed.Query); err != nil {
-			return vterrors.Wrapf(err, "failed creating sentry table")
-		}
-		e.updateMigrationStage(ctx, onlineDDL.UUID, "sentry table created: %s", sentryTableName)
 
+		preparation := func() error {
+			preparationsConn, err := e.pool.Get(ctx, nil)
+			if err != nil {
+				return vterrors.Wrapf(err, "failed getting preparation connection")
```

Comment:
This should just be Wrap. The linters are starting to get more strict about this. Same thing for the other instances of Wrapf w/o any format options/params. I know it's nitty, but otherwise people with newer linters locally will encounter the warnings/errors and have to fix them (e.g. simply when merging in the latest from `main`).

Reply:
Fixed (in two places).